lib/portlayer/storage/image/cache/cache.go

```diff
@@ -507,3 +553,15 @@ func (c *NameLookupCache) DeleteBranch(op trace.Operation, img *image.Image, kee
 
 	return deletedImages, nil
 }
+
+func (c *NameLookupCache) ImageStorageUsage() int64 {
```

Comment:
Rename to GetImageStorageUsage?

Reply:
ImageStorageUsage() is a naming convention of getter in Golang. Please refer to https://golang.org/doc/effective_go.html#Getters